mkosi.images/base/mkosi.conf.d/10-fedora/mkosi.extra/usr/lib/systemd/system/dbus.service

```diff
@@ -0,0 +1 @@
+dbus-broker.service
```

Comment:
@keszybz What's your take on this? To me this seems like something we should solve at the distribution level

Reply:
@richardmaw-codethink `systemctl preset` runs as the first thing during boot so the symlink should be set up before units are loaded.

Reply:
The symlink has been dropped in the latest version, it doesn't appear to be required after all.

I suspect this was a bugfix for a wider issue with a different root cause.
In an earlier version I had accidentally re-installed an older packaged version of systemd over the top of the newer version which could have caused the preset failure problem. The older systemd caused other problems, but I didn't think to check if earlier fixes were still needed.